src/types/plugin-input.ts

```diff
@@ -24,10 +28,33 @@ export const startStopSchema = T.Object({
   ),
   miscellaneous: T.Object(
     {
-      maxConcurrentTasks: T.Number(),
       startRequiresWallet: T.Boolean(),
+      maxConcurrentTasks: T.Array(
+        T.Object({
+          role: userRoleSchema,
+          limit: T.Integer(),
+        })
+      ),
     },
-    { default: { maxConcurrentTasks: 3, startRequiresWallet: true } }
+    {
+      default: {
+        maxConcurrentTasks: [
```

Comment:
These defaults are good and should kick in if the user does not inset any role info into their config.

I think I'd raise "member" to 10 as that's the core contributor/team members and they'd likely be on par with how many issues an admin can take too realistically but @gentlementlegen can confirm these figures.